vuepress/docs/refarch/README.md

```diff
@@ -1,14 +1,362 @@
+
+
+<table>
+  <tr>
+   <td>Bounded Context
+   </td>
+   <td>Purpose
+   </td>
+   <td>Bounded Context
+   </td>
+   <td>Purpose
+   </td>
+  </tr>
+  <tr>
+   <td>Settlements
+   </td>
+   <td>Performs Settlements
+<p>
+Configures settlement models
+<p>
+Calculate Settlements
+   </td>
+   <td>Participant Lifecycle Management
+   </td>
+   <td>Participant Onboarding
+<p>
+Participant Lifecycle Management
+<p>
+Participant Self-service
+<p>
+Participant Self-service UI
+   </td>
+  </tr>
+  <tr>
+   <td>Account Lookup & Discovery
+   </td>
+   <td>Internal Oracle core
+<p>
+Account lookup / discovery
+<p>
+Bulk transactions
+<p>
+Duplicate Identifier management
+<p>
+Inter-scheme connections (incl.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  settlements) cross-border
+   </td>
+   <td>Accounts  & Balances
+   </td>
+   <td>System of record of DFSP participant financial activity & balance
+   </td>
+  </tr>
+  <tr>
+   <td>Transfers & Transactions
+   </td>
+   <td>Transfer processing
+<p>
+Liquidity check for each transfer
+<p>
+Bulk transactions
+<p>
+Multi-currency, incl.multi-hop transactions
+   </td>
+   <td>Agreement (Quoting)
+   </td>
+   <td>Agreement /quoting (core)
+<p>
+Bulk transactions
+<p>
+Multi-currency, incl.multi-hop transactions
+<p>
+Scheme Rule/Patterns Enforcement Happens in each BC
+   </td>
+  </tr>
+  <tr>
+   <td>Scheduling
+   </td>
+   <td>Scheduling time-based events of API calls (Core)
+   </td>
+   <td>Notifications & Alerts
+   </td>
+   <td>Notification state - priority & SLA aware (Core)
+<p>
+Trigger & alert management (Core)
+<p>
+Notifications delivery - priority and SLA aware (Generic)
+   </td>
+  </tr>
+  <tr>
+   <td>FSP Interop APIs
+   </td>
+   <td>ISO External API (Bulk; API, Callbacks triggering (transfers only, Missing in AS-IS)[^6]
+   </td>
+   <td>Third Party Initiated Payments
+   </td>
+   <td>PISP Account Linking
+<p>
+Consent Management
+<p>
+3rd Party Payment Initiation (Core)
+   </td>
+  </tr>
+  <tr>
+   <td colspan="2" >Third party API
+   </td>
+   <td colspan="2" >PISP Mojaloop External API
+<p>
+PISP ISO External API
+   </td>
+  </tr>
+</table>
+
+
+
+### Cross cutting concerns
+
+The following Cross cutting concerns have been identified in Mojaloop:
+
+(This is a high-level description of each Bounded Context.  A more detailed view follows later in this document)
+
+
+<table>
```

Comment:
Table to be converted to markdown --> https://www.markdownguide.org/extended-syntax/#tables

We can try use this tool to convert them --> https://jmalarcon.github.io/markdowntables/